microsoft/testsuites/core/dns.py

```diff
@@ -48,13 +48,28 @@ def verify_dns_name_resolution_after_upgrade(self, node: Node) -> None:
             )
             if 0 != cmd_result.exit_code:
                 node.os.install_packages("unattended-upgrades")
+            if not isinstance(node.os, Ubuntu):
+                if 10 == node.os.information.version.major:
```

Comment:
1. Use version compare pattern like `node.os.information.version >= 10.0.0`.
2. As above pattern, always assume higher version use the same mechanism, until we found it's wrong.
3. Always use an `else` to handle rest versions.

Reply:
updated